portfolio.adobe.com.prod.json

```diff
@@ -5,13 +5,20 @@
   "serviceName":"Portfolio Site",
   "version":1,
   "description":"Enables a domain to work with Adobe Portfolio",
+  "warnPhishing": true,
   "records":[
     {
       "type":"CNAME",
       "host": "www",
       "pointsTo":"@",
       "ttl":3600
     },
+    {
+      "type":"TXT",
+      "host":"@",
+      "data": "%uuid%",
+      "ttl":3600
```

Comment:
us a ttl of 1 hr what we want here? maybe 5 min instead?

Reply:
Mainly was just matching what we agreed on for the other items in the template. Do you think the TXT record should have a different TTL than the other items?

Reply:
do you see domains changing ownership more than once an hour, if not, then this ttl is fine